### ISSUE TITLE
[Suggestion] Config foodValueOverrides saturation value should just be saturation, not a multiplier of hunger

### ISSUE THREAD

**Issue by zhouyu47** [capnkirok/animaniamod#71]
### Expected Behavior
When setting a the foodValueOverrides config, the saturation value should reflect 1:1 the actual saturation that will be used, instead of multiplying the given number by the hunger value and using that instead.

Example:
```    
foodvalues {
    # Food Value Overrides
    S:foodValueOverrides <
        animania:cheese_omelette(5,1)
    >
}
```
Should result in 5 hunger, 1 saturation.

### Actual Behavior
`animania:cheese_omelette(5,1)` results instead in 5 saturation, as it's taking the hunger (5) and multiplying it by the number given for saturation (1) to get it's result.

This is really counter intuitive, and makes certain desired integer values hard to achieve.

### Version of Minecraft, version of Animania
1.12

**Comment by Deshiba** [comment]
I agree

**Comment by stray-sparks** [comment]
I'm really glad you posted this because I had no idea the values needed to be set this way. I was pulling my hair out trying to figure out what numbers to enter to get the values I wanted. I agree they should be made absolute values.

**Comment by Tschipp** [comment]
Yes I also agree. Should be an easy change and should have been changed ages ago.

**Comment by OracleofWuffing** [comment]
Came here because I noticed how high of a saturation bottles of milk were giving out (32 saturation, one bottle of milk is more saturating than two golden carrots 😄 ) so thanks, terrapin47, for pointing out that Animania had the foresight to include the override option in its config file.

I know this is an old issue, but something that I'd like to clarify is that Forge (and I'm guessing Minecraft itself) just uses the saturation of an ItemFood as a multiplier, in all instances of foods- the "Saturation Value" that you program when you make an ItemFood is _not_ the "Saturation" you give the player when the food is eaten.  Instead, the amount of hunger the food heals is multiplied by two, and then _that_ number is multiplied by the "Saturation Value" to get the "Saturation" given to the player.

So if you wanted a food that replenishes 4 hunger (2 shanks) of fullness and replenishes 4 fullness, the Saturation Value would need to be 0.5 to "cancel out" the automatic x2 multiplier.  And I'm guessing that some other foods were designed without taking that process in to mind.

Oooor you can keep Milk and other foods as low hunger, incredible saturation foods.  Milk's tasty.  Nothing wrong with milk. 🥛

**Comment by Tschipp** [comment]
Looking through this, I have decided against it, because that's actually how food saturation is defined also for vanilla food, as a multiplier of the hunger restoration. I wiill however be augmenting the saturation for all animania food, as it is just way too high.
